flow/SystemMonitor.cpp

```diff
@@ -61,6 +67,39 @@ SystemStatistics getSystemStatistics() {
 	    .detail("ApproximateUnusedMemory" #size, FastAllocator<size>::getApproximateMemoryUnused())                    \
 	    .detail("ActiveThreads" #size, FastAllocator<size>::getActiveThreads())
 
+SystemStatistics customSystemMonitorNetworkBusyness(std::string eventName, StatisticsState* statState) {
+	// Custom system monitor to update the network busyness on a 1s cadence
+	const IPAddress ipAddr = IPAddress();
+	SystemStatistics currentStats = getSystemStatistics("", &ipAddr, &statState->systemState, true);
+	NetworkData netData;
+	netData.init();
+	if (!g_network->isSimulated() && currentStats.initialized) {
+		{
```

Comment:
Shouldn't need this extra scope here since it's already in the if statement scope.